addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -969,23 +1081,38 @@
             <field name="children_tax_ids" eval="[(6,0,[ref('tax_ust_7_purchase_13a_auslagerung_skr03'), ref('tax_vst_7_purchase_13a_auslagerung_skr03')])]"/>
             <field name="tax_group_id" ref="tax_group_0"/>
         </record>
+        <record id="fiscal_position_domestic_skr03" model="account.fiscal.position.template">
+            <field name="sequence">1</field>
+            <field name="name">Geschäftspartner Inland</field>
+            <field name="chart_template_id" ref="l10n_de_chart_template"/>
+            <field name="auto_apply" eval="True" />
+            <field name="country_id" ref="base.de"></field>
+        </record>
         <record id="fiscal_position_non_eu_partner_service_skr03" model="account.fiscal.position.template">
+            <field name="sequence">6</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">Dienstleister Ausland</field>
+            <field name="name">Dienstleister Ausland (Nicht-EU)</field>
         </record>
         <record id="fiscal_position_non_eu_partner_skr03" model="account.fiscal.position.template">
+            <field name="sequence">5</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">Geschäftspartner Ausland</field>
+            <field name="name">Geschäftspartner Ausland (Nicht-EU)</field>
         </record>
         <record id="fiscal_position_eu_vat_id_partner_skr03" model="account.fiscal.position.template">
+            <field name="sequence">2</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
             <field name="name">Geschäftspartner EU (mit USt-ID)</field>
+            <field name="auto_apply" eval="True" />
+            <field name="country_group_id" ref="base.europe"></field>
         </record>
         <record id="fiscal_position_eu_vat_id_partner_service_skr03" model="account.fiscal.position.template">
+            <field name="sequence">3</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
             <field name="name">Dienstleister EU (mit USt-ID)</field>
+            <field name="vat_required" eval="True" />
```

Comment:
needs auto_apply True too?

Reply:
we already have one auto applied fiscal position for the partners from a eu country with a valid VAT-ID. This is the tax for the same kind of partners (vat_id is valid, partner from group europe)if they deliver or receive services instead of goods. If i understand correct Odoo doesn't support auto-detection by country-group / country and / or another attribute like (physical goods / service products). Would be good if that could be integrated in a future version as another improvement.

Reply:
The fiscal positions have the possibility to filter on country group.  The difference between physical or service products doesn't require different fiscal positions.  You just put multiple lines in the fiscal position.  I don't see what you could not do.

Reply:
The tax mapping is different, therefore i decided to create a separate fiscal position for services (EU tax case).

Reply:
This makes it more difficult indeed.